artist_query/lib/artist_query.ex

```diff
@@ -0,0 +1,41 @@
+defmodule ArtistQuery do
+  def main(_args) do
+    artist = prompt
+    load_songs("music.csv")
+    |> Enum.filter_map(&(String.upcase(&1["Artist"])) == artist, &(&1["Name"]))
+    |> put_songs(titlecase(artist))
+
+    main([]) # Repeat forever
+  end
+
+  defp put_songs(songs, artist) do
```

Comment:
I think you mean `puts` (short for put string), but I think `print_songs` or `display_songs` would read better.